ext/drreg/drreg.c

```diff
@@ -2478,7 +2478,7 @@ drreg_init(drreg_options_t *ops_in)
         return DRREG_ERROR_INVALID_PARAMETER;
 
     /* Instead of allowing only one drreg_init() and all other components to be
-     * passed in scratch regs by a master, which is not always an easy-to-use
+     * passed in scratch regs by a main, which is not always an easy-to-use
```

Comment:
Hmm to me it sounds off -- the original seems off too.  I would do `s/by a main//`

Reply:
Done.